airflow/values.yaml

```diff
@@ -0,0 +1,191 @@
+# Default values for airflow-helm
+
+replicas:
+  web: 1
+  worker: 1
+  flower: 1
+  scheduler: 1
+
+labels:
+  node_selector_key: openstack-control-plane
+  node_selector_value: enabled
+
+images:
+  airflow: docker.io/eanylin/airflow-helm:1.8.1-1.6.2-0.8
```

Comment:
just as an aside -- i've pushed this image to quay.  you can replace this with `quay.io/attcomdev/airflow-helm:1.0.1`